org.civicrm.bootstrapcivicrm/scss/civicrm/vacancies/_vacancies.scss

```diff
@@ -152,4 +152,13 @@
       }
     }
   }
+
+  .collapsible-title {
```

Comment:
This seems a very generic style. Are you sure it should be relegated to the vacancies page only, and not be move in the `common/` folder instead?